queries.sql

```diff
@@ -23,3 +23,71 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT name, species FROM animals;
+
+COMMIT;
+
```

Comment:
- [ ] As mentioned previously please make sure to verify that the changes have persisted after committing at line `37` you can easily do so by using the SQL statement you have at line `35`.